### PR TITLE
Add support for snowflake dialect

### PR DESCRIPTION
**Pull request by scgkiran**

### CODE REVIEW

**Comment by jacques-n** [comment]
@richtia can you approve checks and review?

**Comment by richtia** [comment]
Thanks for the PR.  Mostly looks good to me!  The test site failure is related to the division by zero in the dialect yaml, which you've marked as a TODO.  I'm fine with fixing that in another PR, since it may involve changes in some of the other dialects as well.

Could you add some information to the README about the new dialect addition?

File: bft/testers/snowflake/config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+snowflake:
+  username: <username>
+  account: <account name>
+  warehouse: <warehouse name>
+  database: <database name>
+  schema: <schema name>
```
Comment (richtia) [inline bft/testers/snowflake/config.yaml:6]: nit: missing EOF newline

**Comment by scgkiran** [comment]
@richtia Thanks for the review. Fixed the build site error and config.yaml. Updated readme. Please take a look.

**Review by richtia — APPROVED** [review]

**Pull request merged**